aidants_connect_erp/admin.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from django.contrib.admin import ModelAdmin
+
+from import_export.admin import ImportMixin
+
+from aidants_connect.admin import VisibleToAdminMetier, admin_site
+
+from .models import CardSending
+
+logger = logging.getLogger()
+
+
+class CardSendingAdmin(ImportMixin, VisibleToAdminMetier, ModelAdmin):
+    list_display = (
+        "sending_date",
+        "organisation",
+        "quantity",
+        "status",
+        "command_number",
+    )
+    list_filter = ("status",)
+
+    raw_id_fields = ("organisation",)
+
+
+admin_site.register(CardSending, CardSendingAdmin)
```

Comment:
Tu peux aussi utiliser [le décorateur @register](https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#the-register-decorator) si tu veux :

```python
@admin.register(CardSending, site=admin_site)
```

Reply:
ouaip mais j'avoue que j'aime bien mes petits habitudes de la déclaration à la fin pour avoir tout au même endroit quand tu registre juste un modele sans model admin spécifique et quand tu registres les models admin en plus. 

Comme ça un seul et unique endroit pour tout

Reply:
👍